src/parser/vast_parser.js

```diff
@@ -98,30 +99,53 @@ export class VASTParser extends EventEmitter {
    * Returns a Promise which resolves,rejects according to the result of the request.
    * @param  {String} url - The url to request the VAST document.
    * @param {Number} wrapperDepth - how many times the current url has been wrapped
-   * @param {String} originalUrl - url of original wrapper
+   * @param {String} previousUrl - url of the previous VAST
    * @emits  VASTParser#VAST-resolving
    * @emits  VASTParser#VAST-resolved
    * @return {Promise}
    */
-  fetchVAST(url, wrapperDepth, originalUrl) {
+  fetchVAST(url, wrapperDepth = 0, previousUrl = null) {
     return new Promise((resolve, reject) => {
       // Process url with defined filter
       this.URLTemplateFilters.forEach(filter => {
         url = filter(url);
       });
 
       this.parentURLs.push(url);
-      this.emit('VAST-resolving', { url, wrapperDepth, originalUrl });
+      const timeBeforeGet = Date.now();
```

Comment:
Nope. I had it at first but actually Node doesn't support it natively and it is not standard afaik. If you actually look at all timing calculations in vast-client, we always use Date.now because of this